build.gradle

```diff
@@ -3,16 +3,16 @@ plugins {
 }
 
 group 'io.confluent.oauth'
-version '1.0-SNAPSHOT'
+version '1.1-SNAPSHOT'
 
 repositories {
     mavenCentral()
 }
 
 dependencies {
-    implementation group: 'org.apache.kafka', name: 'kafka-clients', version: '3.3.1'
-    implementation 'com.fasterxml.jackson.core:jackson-databind:2.13.3'
-    implementation 'org.bitbucket.b_c:jose4j:0.7.9'
+    implementation group: 'org.apache.kafka', name: 'kafka-clients', version: '3.6.1'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.14.0-rc1'
```

Comment:
Any reason for needing an RC version of `jackson-databind` here - do we depend on something in the RC, or could we use a latest GA release?